mmv1/products/networkservices/GrpcRoute.yaml

```diff
@@ -0,0 +1,247 @@
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/traffic-director/docs/reference/network-services/rest/v1beta1/projects.locations.grpcRoutes'
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 30
+      update_minutes: 30
+      delete_minutes: 30
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+autogen_async: true
+import_format: ['projects/{{project}}/locations/global/grpcRoutes/{{name}}']
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    min_version: beta
+    name: 'network_services_grpc_route_basic'
+    primary_resource_id: 'default'
+    vars:
+      resource_name: 'my-grpc-route'
+  - !ruby/object:Provider::Terraform::Examples
+    min_version: beta
+    name: 'network_services_grpc_route_matches_and_actions'
+    primary_resource_id: 'default'
+    vars:
+      resource_name: 'my-grpc-route'
+  - !ruby/object:Provider::Terraform::Examples
+    min_version: beta
+    name: 'network_services_grpc_route_actions'
+    primary_resource_id: 'default'
+    vars:
+      resource_name: 'my-grpc-route'
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    required: true
+    url_param_only: true
+    immutable: true
+    description: |
+      Name of the GrpcRoute resource.
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'selfLink'
+    description: |
+      Server-defined URL of this resource.
+    output: true
+  - !ruby/object:Api::Type::Time
+    name: 'createTime'
+    description: |
+      Time the GrpcRoute was created in UTC.
+    output: true
+  - !ruby/object:Api::Type::Time
+    name: 'updateTime'
+    description: |
+      Time the GrpcRoute was updated in UTC.
+    output: true
+  - !ruby/object:Api::Type::KeyValuePairs
+    name: 'labels'
+    description: Set of label tags associated with the GrpcRoute resource.
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      A free-text description of the resource. Max length 1024 characters.
+  - !ruby/object:Api::Type::Array
+    name: hostnames
+    item_type: Api::Type::String
+    description: |
+          Required. Service hostnames with an optional port for which this route describes traffic.  
+    required: true
+    send_empty_value: true  
+  - !ruby/object:Api::Type::Array
+    name: meshes
+    item_type: Api::Type::String
+    description: |
+          List of meshes this GrpcRoute is attached to, as one of the routing rules to route the requests served by the mesh.  
+    send_empty_value: true 
```

Comment:
Why do we need all these send_empty_value: true?

Reply:
You are right, we dont need them. Fixed.